### PR TITLE
Adds the bigswitch package install

### PR DESCRIPTION
**Pull request by marios**
grab the rpms if they aren't in local directory and then use
them to also virt-customize install the rpm and enable send-lldp

### CODE REVIEW

File: bigswitch-patch-image.sh [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 set -euxo pipefail
 
+IMAGE_PATH=${1:-"/home/stack/overcloud-full.qcow2"}
```
Comment (xinwu) [inline bigswitch-patch-image.sh]: According to https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux_OpenStack_Platform/7/html/Director_Installation_and_Usage/sect-Creating_Directories_for_Templates_and_Images.html, the images are at /home/stack/images, Let's follow the same pattern.

**Comment by xinwu** [comment]
Minor comments. Otherwise looks good :)

File: bigswitch-patch-image.sh (hunk 1 above)
Comment (marios) [inline bigswitch-patch-image.sh]: thanks good point will do :)

**Comment by marios** [comment]
thanks for looking Xin. FYI I am having some luck with 

```
 virt-customize -a overcloud-full.qcow2 --firstboot install_bigswitch_packages.sh 
```

where the firstboot script is simply like

```
rpm -ivh --force /root/python-networking-bigswitch-2015.1.37-1.el7.centos.noarch.rpm
rpm -ivh --force /root/openstack-neutron-bigswitch-lldp-2015.1.37-1.el7.centos.noarch.rpm;
systemctl enable neutron-bsn-lldp.service
```

(the --force here is because it was already installed on that image from the earlier invocation of virt-customize as in the script above)

I will test again from scratch for sanity check and update this review with whatever works before we merge

**Comment by marios** [comment]
-1 DO NOT MERGE (will update with comments above in a bit)

**Comment by marios** [comment]
Latest version uses first boot commands instead and fixes the server startup issues we see at https://bugzilla.redhat.com/show_bug.cgi?id=1258106 and includes comment from Xin.

IMO we can land this and iterate when necessary

**Comment by jistr** [comment]
:+1:

**Pull request merged**